pegasus/sites.v3/code.org/public/learn2016/index.haml

```diff
@@ -14,6 +15,34 @@ social:
 
 = view 'mobilecheck.js'
 
+#fullwidth{style: "background-color: #2799a4; overflow: hidden"}
+  .thegap{style: "height: 1px"}
+    &nbsp;
+  #desktop-header.desktop-feature
+    = view :header, variant: "desktop"
+  #mobile-header.mobile-feature
+    = view :header, variant: "mobile"
+
+  .banner
```

Comment:
Might refactor this (including hoisting CSS) once we get banners for robotics/unplugged variant pages.